us-enrichment-api/lookup.go

```diff
@@ -23,6 +24,45 @@ type enrichmentLookup interface {
 	populate(query url.Values)
 }
 
+type genericLookup struct {
+	Lookup     *Lookup
+	DataSet    string
+	DataSubset string
+	Response   []byte
+}
+
+func (g *genericLookup) getSmartyKey() string     { return g.Lookup.SmartyKey }
+func (g *genericLookup) getDataSet() string       { return g.DataSet }
+func (g *genericLookup) getDataSubset() string    { return g.DataSubset }
+func (g *genericLookup) getLookup() *Lookup       { return g.Lookup }
+func (g *genericLookup) getResponse() interface{} { return g.Response }
+func (g *genericLookup) unmarshalResponse(bytesResponse []byte, headers http.Header) error {
```

Comment:
To keep consistency with the rest of the lookup structs, lets change the bytesResponse variable name to bytes.